tests/components/tplink/test_device_tracker.py

```diff
@@ -1,13 +1,14 @@
 """The tests for the tplink device tracker platform."""
 
 import os
+
```

Comment:
If you want to clean up the device_tracker tests, please create another PR for those. These are irrelevant for the issue at hand.